### PR TITLE
generate annotations in yml format

### PR DESCRIPTION
**Pull request by dbu**
replaces #568

i went with yml rather than annotations. what would annotations do, service definitions by annotations?

### CODE REVIEW

File: tutorial/getting-started.rst [hunk 1]
```diff
@@ -427,7 +427,7 @@ and add some posts::
         }
     }
 
-The 
+The
```
Comment (dbu) [inline tutorial/getting-started.rst]: this was already broken in the original commit: 6f21e4bf4e09b46e5050f29e9786d9f8ea382acb  - i tracked further back and did not find where the The appeared. just dropping it.

**Comment by dbu** [comment]
okay, updated

**Comment by dbu** [comment]
ping @WouterJ

**Comment by WouterJ** [comment]
This should be merged into 1.1, I believe?

(we should create a plan about when to deprecate versions)

**Comment by dbu** [comment]
agreed. i can merge on cli if you want

**Comment by dbu** [comment]
if its all good, that is

**Comment by WouterJ** [comment]
Thanks David.

**Pull request closed without merging**